akka-sample-persistence-dc-java/build.sbt

```diff
@@ -33,4 +33,7 @@ libraryDependencies ++= Seq(
   "org.scalatest" %% "scalatest" % "3.0.7" % "test"
 )
 
+// transitive dependency of akka 2.5x that is brought in by addons but evicted
+dependencyOverrides += "com.typesafe.akka" %% "akka-protobuf" % AkkaVersion
+
```

Comment:
add mvn pom while we are at it, and update run instructions in readme
template commit: https://github.com/akka/akka-samples/pull/141/commits/b54f1b608d38d4f1c49d6df2a023a8f4d256f5e5

Reply:
:+1: added